docs/learn/tokenomics2/Inflation.md

```diff
@@ -2,33 +2,166 @@
+
+Regular staker rewards are awarded for staking native currency, **ASTR**, on a dApp.
+These rewards have two components - the _base_ reward and the _adjustable_ reward.
+
+Base reward is the amount assigned to the reward pool regardless of how much has been staked in total.
+
+$base\_staker\_reward\_pool\_per\_era = \frac{total\_base\_staker\_cycle\_reward}{number\_of\_cycles * eras\_per\_build\_and\_earn}$
+
+The adjustable part is the dynamic part, and depends on the _total value staked_ and the _target stake value_.
+This amount linearly increases as the _total value staked_ increases, and then saturates once the amount is reached or exceeded.
+With this component, _staker rewards_ are not a _zero-sum game_.
+
+$max_\_adjustable\_staker\_reward\_pool\_per\_era = \frac{total\_adjustable\_staker\_cycle\_reward}{number\_of\_cycles * eras\_per\_build\_and\_earn}$
+
+The adjustable part of the reward is calculated once an era ends, using the _total value staked_ at that point in time.
+
+$adjustable\_factor = min(1, \frac{total\_value\_staked\_percent}{ideal\_staking\_percent})$
+
+Using the _adjustable\_factor_, adjustable portion of the staker reward is:
+
+$adjustable\_staker\_reward\_pool = max_\_adjustable\_staker\_reward\_pool\_per\_era * adjustable\_factor$
+
+When the _adjustable factor_ is less than **1**, it means the remainder is never minted, reducing the overall inflation.
+
+With the above formulas, we can finally express how much staker _Alice_ earns in era **n**:
+
+$staker\_reward_{Alice} = \frac{staked\_value_{Alice,n}}{total\_staked\_value_n} * (base\_staker\_reward\_pool\_per\_era_n + adjustable\_staker\_reward\_pool_n)$
+
+#### Bonus Rewards
+
+In case a staker stakes during the `Voting` subperiod, and doesn't reduce their stake below what was staked at the end of the `Voting` subperiod,
+it will make them eligible for the bonus reward.
+
+Bonus reward pool is assigned per period, and can be expressed as:
+
+$bonus\_reward\_pool\_per\_period = \frac{total\_bonus\_cycle\_reward}{number\_of\_cycles}$
+
+The bonus reward for a staker _Alice_ can then be expressed as:
+
+$bonus\_staker\_reward_{Alice} = \frac{voting\_subperiod\_staked\_value_{Alice}}{total\_voting\_subperiod\_staked\_value} * (bonus\_reward\_pool\_per\_period)$
+
+## Lazy Minting
+
+Both _staker_ and _dApp_ rewards are minted in a lazy fashion - when they are needed. Only collator & treasury rewards are minted per block.
+
+With the _adjustable staker reward_ and the dApp staking tier system, the inflation in practice will be much lower than the calculated _soft-cap_.
+We only allow _award_ the full extent of the adjustable staker rewards if we've reached or exceeded the ideal staking rate.
+For the dApp rewards, it's unlikely that all of the tiers will be filled with dApps - it might be that the tier capacity is larger than the demand,
+or that simply some dApps don't attract enough support to enter a tier. These rewards will never be even minted.
```

Comment:
These rewards will never be even minted thus lowering the effective inflation rate.

Reply:
Done